### DISCUSSION TITLE
Best Version and settings for RTX 3060 with LHR to miner

### DISCUSSION THREAD

**Discussion by cryptonitaproject** [NebuTech/NBMiner#508] [Q&A]
Hi, I try most settings for RTX 3060 with LHR, I try 39.3 , 39.4 and 39.5 versions and I dont have good results, the last good version for me is 39.2, the latest version I dont get best results.
Please can you tell me yours configurations? do you have best results? my hashrate with each card is 35 MH/s
thanks

**Comment by ssoczek** [comment]
I use 39.5 and i have 39,8

**Comment by JDDelgado** [comment]
hola, yo manejo power limit 80, core clock en -502, memory clock x1615, y obtengo de 36 a 37mhs estables, temperatura de 57g con buena ventilación 98w de consumo con esa 3060 lhr v2
